include/cuco/detail/utils.cuh

```diff
@@ -102,5 +102,112 @@ struct strong_type {
   T value;  ///< Underlying value
 };
 
+/**
+ * @brief Rounds `v` to the nearest power of 2 greater than or equal to `v`.
+ *
+ * @param v
+ * @return The nearest power of 2 greater than or equal to `v`.
+ */
+constexpr std::size_t next_pow2(std::size_t v) noexcept
+{
+  --v;
+  v |= v >> 1;
+  v |= v >> 2;
+  v |= v >> 4;
+  v |= v >> 8;
+  v |= v >> 16;
+  return ++v;
+}
```

Comment:
This can be replaced with `cuda::std::bit_ceil` from the `<cuda/std/bit>` header.